frontend/packages/dev-console/src/components/import/git/__tests__/GitSection.spec.tsx

```diff
@@ -0,0 +1,89 @@
+import * as React from 'react';
+import { render, waitFor, screen, configure, cleanup, fireEvent } from '@testing-library/react';
+import { Formik, FormikConfig } from 'formik';
+import { act } from 'react-dom/test-utils';
+import { Provider } from 'react-redux';
+import { GitProvider } from '@console/git-service/src';
+import store from '@console/internal/redux';
+import { devStrategyInput } from '../../../../../integration-tests/testData/git-import-flow.data';
+import userEvent from '../../../buildconfig/__tests__/user-event';
+import DevfileStrategySection from '../../devfile/DevfileStrategySection';
+import { GitSectionFormData } from '../GitSection';
+
+configure({ testIdAttribute: 'data-test-id' });
+
+const Wrapper: React.FC<FormikConfig<GitSectionFormData>> = ({ children, ...formikConfig }) => (
+  <Provider store={store}>
+    <Formik {...formikConfig}>
+      {(formikProps) => (
+        <form onSubmit={formikProps.handleSubmit}>
+          {children}
+          <input type="submit" value="Submit" />
+        </form>
+      )}
+    </Formik>
+  </Provider>
+);
+
+describe('DevFileImportStrategy', () => {
```

Comment:
Doesn't this test file primary test `DevfileStrategySection` ? In this case the file should also renamed and moved into `../import/devfile/__tests__` folder and should be called `DevfileStrategySection.spec.tsx` or?

```suggestion
describe('DevfileStrategySection', () => {
```